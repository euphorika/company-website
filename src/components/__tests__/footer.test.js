import React from "react"
import { shallow } from "enzyme"
import toJson from 'enzyme-to-json'
import { useStaticQuery } from "gatsby"

import Footer from "../footer"

beforeEach(() => {
  useStaticQuery.mockReturnValueOnce({
    site: {
      siteMetadata: {
        email: 'me@example.com',
        phone: '1234567-890'
      }
    }
  })
})

describe("Footer", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
