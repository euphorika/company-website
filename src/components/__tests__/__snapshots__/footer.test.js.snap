// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="pageFooter"
>
  <ul>
    <li>
      <a
        href="/legal-notice/"
      >
        Impressum
      </a>
    </li>
    <li>
      <a
        href="/data-protection-and-privacy/"
      >
        Datenschutz
      </a>
    </li>
  </ul>
</footer>
`;
